### ISSUE TITLE
Change 'group' => t('User') to 'group' => t('User account')?

### ISSUE THREAD

**Issue by olafgrabienski** [backdrop-contrib/rules#196]
When I recently ported the [Rules Block/Unblock User](https://github.com/backdrop-contrib/rules_block_user) module from D7 to Backdrop, I changed the name of the user account group for the "React on event" dropdown from `'group' => t('User')` to `'group' => t('User account')`. I was doing this because the term "User" in Backdrop was renamed to "User account" at some point.

Using my new module, I noticed the term "User" wasn't renamed (at all places) in Rules. The result is a "React on event" dropdown where most items are grouped under "User" but some under "User account" (see screenshot below). Can we change the group name in Rules to "User account"?

**Code example**
https://github.com/backdrop-contrib/rules/blob/8bfbe865a4355b85adf2d1492a07c9e98bb07e54/modules/user.rules.inc#L25

**Screenshot**
Rules "React on event" dropdown on a site with the *Rules Block/Unblock User* module

![backdrop-rules-react-on-event](https://github.com/backdrop-contrib/rules/assets/14188307/35e71dcb-ee12-4dbe-af67-83994916edc8)

**Comment by bugfolder** [comment]
This makes sense to me. @laryn, @argiepiano?

**Comment by laryn** [comment]
Agreed. Thanks @olafgrabienski!

**Comment by argiepiano** [comment]
Seems reasonable!

**Comment by bugfolder** [comment]
I'll work on this. There's more than just changing the the group; there's lots of places where the language needs changing from "user" to "user account" (or "person", if we're talking about the actual user), so there will be a bunch of little tweaks in the PR.

**Comment by bugfolder** [comment]
Changes made. All tests pass except the two that previously weren't passing: "Rules UI minimal profile tests" and "Rules I18n".

**Comment by olafgrabienski** [comment]
Thanks for the PR, @bugfolder, it looks good to me, with one exception:

I'm not sure if "user account" is a convenient term in the three phrases below. In my understanding, it's always a person (or a bot) who logs in, not an account. Not sure about good alternatives. Mayby stick with "user" in these cases?

line 74:
```
- 'label' => t('User has logged in'),
+ 'label' => t('User account has logged in'),
```

line 77:
```
- 'account' => array('type' => 'user', 'label' => t('logged in user')),
+ 'account' => array('type' => 'user', 'label' => t('logged in user account')),
```

line 82:
```
- 'label' => t('User has logged out'),
+ 'label' => t('User account has logged out'),
```

**Comment by argiepiano** [comment]
> I'm not sure if "user account" is a convenient term in the three phrases 

Yes, it's one of the limitations of using "user account" instead of "user" when you have action verbs like "log in". The alternative is a bit awkward. "Person has logged in"? Never seen that. "People have logged in"? Confusing. If we are going to be inexorable about avoiding "User" , then "User account has logged in" is probably the least bad option.

LGTM  (left a comment - please check)

**Comment by bugfolder** [comment]
I started looking through core for user-facing (person-facing?) text that talks about logged-in beings. Most examples use "user," e.g., in user.tokens.inc:

```
    'description' => t('Tokens related to the currently logged in user.'),
...
    'description' => t('The date the user last logged in to the site.'),
```

But I did find in user.views.inc,

```
        'title' => t('User account ID from logged in person'),
```

(There's lots of "logged-in user" in code comments, too.)

I didn't see any "logged-in user account," so those should probably be changed (back) in this PR. Since core seems to still be mostly using "logged in user," it would probably be safest in these three cases to go back to "user".
